src/mod/dns.mod/coredns.c

```diff
@@ -41,6 +41,13 @@
 #include <resolv.h>
 #include <errno.h>
 
+/* OpenBSD */
```

Comment:
what about other systems? NS_GET16 doesn't seem to be documented, is it portable?

Reply:
Thank you for your interesting question.

glibc added NS_GET16 with https://github.com/lattera/glibc/commit/b43b13ac2544b11f35be301d1589b51a8473e32b#diff-bfb189ccea33893d61012399248a574f in 1999. Before, it used GETSHORT, so the OpenBSD compat code here also compensates for old glibc.

FreeBSD added NS_GET16 with https://github.com/freebsd/freebsd/commit/f8f1d6e80f37646e814e2fad78e9a39580b7fbf9#diff-1dfdc4f29f86e1a82e30a69222cf8fd1 in 1999. Again, before, it used GETSHORT.

NetBSD added NS_GET16 with http://cvsweb.netbsd.org/bsdweb.cgi/src/include/arpa/nameser.h.diff?r1=1.15&r2=1.16&only_with_tag=MAIN&f=h in 2004.  Again, before, it used GETSHORT.

Im now checking more systems and histories...